### PR TITLE
zh-translation: boilerplates/before-you-begin-egress.md

### PR DESCRIPTION
**Pull request by elkan1788**
Upgrade the Chinese translation in path:  boilerplates/before-you-begin-egress.md

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by SataQiu — COMMENTED** [review]
/ok-to-test

File: content/zh/boilerplates/before-you-begin-egress.md [hunk 1]
```diff
@@ -4,7 +4,11 @@
 
 *   按照[安装指南](/zh/docs/setup/)中的说明安装 Istio。
 
-*   启动 [sleep]({{< github_tree >}}/samples/sleep) 示例，以获取发送请求的测试源。
+    {{< tip >}}
+    如果您安装`demo`的[安装配置](/zh/docs/setup/additional-setup/config-profiles/)，则将启用出口网关和访问日志。
```
Comment (kebe7jun) [inline content/zh/boilerplates/before-you-begin-egress.md:8]:
```suggestion
    如果您安装`demo`的[安装配置](/zh/docs/setup/additional-setup/config-profiles/)，则将启用 Egress Gateway和访问日志。
```

Comment (elkan1788) [reply content/zh/boilerplates/before-you-begin-egress.md:8]: Thank you for review.

Comment (kebe7jun) [reply content/zh/boilerplates/before-you-begin-egress.md:8]: 修改这个之后就能合了哟。

Comment (elkan1788) [reply content/zh/boilerplates/before-you-begin-egress.md]: 哈哈，忘记PUSH一下，貌似最近外网访问都有点小问题。

**Review by kebe7jun — APPROVED** [review]
lgtm

**Pull request merged**